app/src/main/java/com/andela/art/settings/SettingsActivity.java

```diff
@@ -21,47 +24,32 @@
 /**
  * This is activity manages a users settings.
  */
-public class SettingsActivity extends AppCompatActivity {
+public class SettingsActivity extends AppCompatActivity implements Dialog.OnClickListener {
     @Inject GoogleSignInClient googleSignInClient;
+    SettingsPageBinding settingsPageBinding;
```

Comment:
This can be made a local variable in `onCreate`

Reply:
Thanks. I have made it a local variable.